From 1a8d0dee3e619f988f501cd4c9bd0775dd4590fc Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Wed, 14 May 2025 00:05:05 +0300
Subject: [PATCH] [SW-227599] gaudi2: interrupt_info: add reset req

For handling BIS violation by requesting reset (similar to gaudi3)

Signed-off-by: Guy Nisan <gnisan@habana.ai>
Change-Id: Ic4a3924a3f0435c937fc7a88a0f06e4b51c90a82
---

diff --git a/drivers/misc/hl_npu/gaudi2/interrupt_info.c b/drivers/misc/hl_npu/gaudi2/interrupt_info.c
index ca53947..7813803 100644
--- a/drivers/misc/hl_npu/gaudi2/interrupt_info.c
+++ b/drivers/misc/hl_npu/gaudi2/interrupt_info.c
@@ -1657,6 +1657,7 @@
 		break;
 
 	case GAUDI2_EVENT_CPU_FP32_NOT_SUPPORTED:
+	case GAUDI2_EVENT_CPU_DEV_RESET_REQ:
 		hl_gaudi2_submit_eq_entry(event, evt_id, dev);
 		if (hl_sec_is_enabled(dev))
 			hl_request_self_reset(dev);
